sdk/appcenter-data/src/test/java/com/microsoft/appcenter/data/DataListTest.java

```diff
@@ -143,11 +132,12 @@ public void listPrimitiveTypeWhenThereArePendingOperations() {
                 FUTURE_TIMESTAMP,
                 CURRENT_TIMESTAMP,
                 CURRENT_TIMESTAMP);
-        List<LocalDocument> storedDocuments = new ArrayList<>();
-        storedDocuments.add(localDocument);
-        storedDocuments.add(expiredDocument);
-        storedDocuments.add(notPendingDocument);
-        storedDocuments.add(notPendingNotExpiredDocument);
+        List<LocalDocument> storedDocuments = new ArrayList<LocalDocument>() {{
```

Comment:
FYI this creates a sub class of ArrayList and is not really improving anything.
Ok for test, but keep in mind not to use this pattern in SDK code.

Reply:
Ah, good to know! I thought it was just for readability like those initializers in C#.